CONTRIBUTING_TO_OPENSHIFT_PLUGINS.md

```diff
@@ -0,0 +1,426 @@
+# Development and release process for the OpenShift Jenkins plugins
+
+The OpenShift organization currently maintains several plugins related to integration between V3 OpenShift and V2 Jenkins:
+
+* [OpenShift Client Plugin](https://github.com/openshift/jenkins-client-plugin)
+* [OpenShift Sync Plugin](https://github.com/openshift/jenkins-sync-plugin)
+* [OpenShift Login Plugin](https://github.com/openshift/jenkins-openshift-login-plugin)
+* [OpenShift Pipeline Plugin](https://github.com/openshift/jenkins-plugin)  (*only in maintenance at this point*)
+
+Per the process for [hosting plugins](https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins) specified by the open source Jenkins
+project, there are associated wikis in the Jenkins project and analogous repositories under the `jenkinsci` project on GitHub.  Those repository URLs are:
+
+* [OpenShift Client Plugin](https://github.com/jenkinsci/openshift-client-plugin)
+* [OpenShift Sync Plugin](https://github.com/jenkinsci/openshift-sync-plugin)
+* [OpenShift Login Plugin](https://github.com/jenkinsci/openshift-login-plugin)
+* [OpenShift Pipeline Plugin](https://github.com/jenkinsci/openshift-pipeline-plugin)  (*again, only in maintenance at this point*)
+
+And the wiki URLs are: 
+
+* [https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin)
+* [https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Sync+Plugin)
+* [https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Client+Plugin)
+* [https://wiki.jenkins.io/display/JENKINS/OpenShift+Pipeline+Plugin](https://wiki.jenkins.io/display/JENKINS/OpenShift+Login+Plugin)
+
+As the development process for each of these plugins are similar in many ways, we are documenting the process here under 
+the repository for the OpenShift Jenkins images, where each of the plugins will reference this document in their respective 
+READMEs.
+
+## Constructing your development sandbox
+
+You will need an environment with the following tools installed:
+
+* Maven (the `mvn` command)
+* Git
+* Java
+* an IDE or editor of your choosing
+
```

Comment:
Maybe list the min/max versions of each tool here if applicable?

Reply:
for java, yes will update